datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -911,6 +912,9 @@ async fn roundtrip_expr_api() -> Result<()> {
         count_distinct(lit(1)),
         first_value(lit(1), None),
         first_value(lit(1), Some(vec![lit(2).sort(true, true)])),
+        functions_window::nth_value::first_value(lit(1)),
+        functions_window::nth_value::last_value(lit(1)),
```

Comment:
These expression functions (`first_value` and `last_value`) did not exist before. There is only `nth_value` here:
https://docs.rs/datafusion/latest/datafusion/logical_expr/window_function/index.html#functions

I guess it is alright. 

Thanks for adding them 👍